src/components/PostEditor.js

```diff
@@ -285,14 +280,14 @@ export class PostEditor extends React.PureComponent {
 
   render () {
     const {
-      post, postEdit, dispatch, imagePending, saving, id, defaultTags, placeholder
+      parentPost, post, postEdit, dispatch, imagePending, saving, id, defaultTags, placeholder
     } = this.props
     const { currentUser } = this.context
     const { description, community_ids, tag, linkPreview } = postEdit
     const selectableTags = uniq(compact([this.props.tag, tag].concat(defaultTags)))
     const { name, showDetails } = this.state
     const editorType = this.editorType()
-    const shouldSelectTag = editorType !== 'project'
+    const shouldSelectTag = !includes(['project', 'event'], editorType)
```

Comment:
I think you want to go in the other direction here and remove `shouldSelectTag` entirely, because now all types of posts should show the tag selector dropdown.